app/views/projects/samples/transfers/_transfer_modal.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= viral_dialog(open: open) do |dialog| %>
+  <%= dialog.with_header(title: t("projects.samples.transfers.new.title")) %>
+  <%= dialog.with_section do %>
+    <%= form_with(url: namespace_project_samples_transfer_path, method: :post) do |form| %>
+      <div
+        data-controller="projects--samples--transfers"
+        data-projects--samples--transfers-storage-key-value=<%=namespace_project_samples_url%>
+        data-projects--samples--transfers-target="field"
+        data-projects--samples--transfers-field-name-value="sample_ids[]"
+      />
+      <div class="form-field">
+        <%= form.label :new_project_id %>
+        <%= form.collection_select(:new_project_id, @projects, :id, :full_path) %>
+        <%= render "shared/form/field_errors",
+        errors: @project.errors.full_messages_for(:base) %>
+      </div>
+      <div class="my-2">
```

Comment:
**nitpick:** The button should be a little lower as it looks like it is part of the form item (yes this is very nitpicky).  I have been trying to keep a `4px` spacing between form elements ( I have been playing with `grid grid-col-1 gap-4` and a div containing all form controls see `app/views/projects/_change_path.html.erb`).  As well can you align the buttons to the right?  This is because most of these types of forms use a confirmation component and they align the buttons to the right.  You can see the confirmation modals on the project settings page.

Reply:
Hopefully this is what you're asking for. Please see b9b7d17a51ba907ca4cba0ebca8c662d8405ad4f.

Reply:
That is exactly what I was looking for, thanks you!